### PR TITLE
Update the module name #1390

### LINKED ISSUE #1378: Use vanity urls to change the go module name
**Describe the problem/challenge you have**
Details in https://github.com/carvel-dev/carvel/issues/689

**Describe the solution you'd like**
Use vanity urls to change the go module name in kapp-controller (use imgpkg [PR](https://github.com/carvel-dev/imgpkg/pull/597) as a reference)

**Anything else you would like to add:**

---
Vote on this request

This is an invitation to the community to vote on issues, to help us prioritize our backlog. Use the "smiley face" up to the right of this comment to vote.

👍 "I would like to see this addressed as soon as possible"
👎 "There are other more important things to focus on right now"

We are also happy to receive and review Pull Requests if you want to help working on this issue.

### PR DESCRIPTION
**Pull request by joaopapereira**
#### Which issue(s) this PR fixes:

Fixes #1378 

##### Review Checklist:

- [ ] Follows the [developer guidelines](https://carvel.dev/shared/docs/latest/development_guidelines/)
- [ ] Relevant tests are added or updated
- [ ] Relevant docs in this repo added or updated
- [ ] Relevant carvel.dev docs added or updated in a separate PR and there's
  a link to that PR
- [ ] Code is at least as readable and maintainable as it was before this
  change

#### Additional documentation e.g., Proposal, usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/fetch/vendir.go [hunk 1]
```diff
@@ -13,10 +13,10 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/kappctrl/v1alpha1"
+	"carvel.dev/kapp-controller/pkg/apis/kappctrl/v1alpha1"
 	// we run vendir by shelling out to it, but we create the vendir configs with help from a vendored copy of vendir.
+	"carvel.dev/kapp-controller/pkg/exec"
```
Comment (praveenrewar) [inline pkg/fetch/vendir.go:17]: nit: The comment should be for the vendirconf import

**Review by praveenrewar — APPROVED** [review]
Just one small optional nit, looks good otherwise.

File: pkg/fetch/vendir.go (hunk 1 above)
Comment (joaopapereira) [reply pkg/fetch/vendir.go:17]: fixed

**Review by praveenrewar — APPROVED** [review]

**Comment by joaopapereira** [comment]
When we release 0.52.0 we should merge the PR and create version 0.53.0

**Comment by praveenrewar** [comment]
I was thinking that we should make a release now with this change. We currently have 3 commits after the last release, 2 of them are bumps and one of them is a fix for using kctrl as a library, so ideally one would want to bump to this release for those changes.
We could also wait, but then there's a chance that there will be merge conflicts on this PR and we would have to again spend some time on resolving them.

**Comment by joaopapereira** [comment]
I am ok with releasing it. On Monday if y'll have some time can you release it and merge this PR?

**Comment by praveenrewar** [comment]
Sure, I will do it.

**Pull request merged**